SanctumLabs/ujumbe
# Submitted SMS Consumer

## Description

Submitted SMS Consumer

## How can we test this?

1. Run `docker compose up`
2. Run main application
3. Run consumer application
4. Make API request to send SMS
5. SMS should be pushed to topic and then consumed by consumer application and later on saved to the database before another sms event is published to delver the SMS

## Checklist:
